algorithms/scripts/SGDClassifier-Isomap-KT12.py

```diff
@@ -3,7 +3,7 @@
 
 # # Create a logistic regression model to predict TP53 mutation from gene expression data in TCGA using Isomap as a dimensionality reduction technique
 
-# In[1]:
+# In[27]:
```

Comment:
Sorry about that - must have made another change and re-run the notebook.

Test AUROC went from ~63% to ~75%.